q2cli/tools.py

```diff
@@ -16,9 +16,26 @@ def tools():
     pass
 
 
+@tools.command(name='export', short_help='Export data.',
+               help="Export data from a QIIME Artifact or Visualization.")
+@click.option('--input-path', required=True,
+              type=click.Path(exists=True, dir_okay=True),
+              help='Path to Artifact or Visualization that should be '
+                   'exported.')
+@click.option('--output-path', required=True,
```

Comment:
Just had some discussion about the `qiime tools export` interface and we decided to have it match `tools extract` pretty closely. I'll issue a PR soon with those changes and will address my comment above about `dir_okay=False`.